src/exe/cimbar/cimbar.cpp

```diff
@@ -177,6 +177,7 @@ int main(int argc, char** argv)
 	unsigned compressionLevel = cimbar::Config::compression_level();
 	unsigned ecc = cimbar::Config::ecc_bytes();
 	options.add_options()
+		("n,encode", "Run the encoder!", cxxopts::value<bool>())
```

Comment:
I'm not sure about `-n`, but `-e` is already used for `ecc`. And I've seen unixtools commit worse crimes against the alphabet than this.